integration-test/langfuse-integration-node.spec.ts

```diff
@@ -627,4 +643,15 @@ describe("Langfuse Node.js", () => {
     const sessions = await langfuse.fetchSessions();
     expect(sessions.data).toContainEqual(expect.objectContaining({ id: sessionId }));
   });
+
+  it("create and fetch prompts", async () => {
```

Comment:
please add multiple prompts (name) and multiple versions of the same prompt in this